src/stylesheets/components/_footer.scss

```diff
@@ -272,3 +272,43 @@ li.usa-footer-primary-content {
     padding-left: 1em;
   }
 }
+
+.usa-social-link {
+  $hit-area: 4.5rem; // used for height and width
+  $left-align-adjust: 1.2rem; // relative left positioning
+  $blank-space: 1rem; // spacing between link hit areas
+
+  background-position: center center;
+  background-repeat: no-repeat;
+  display: inline-block;
+  height: $hit-area;
+  left: $left-align-adjust;
+  margin-left: $blank-space;
+  position: relative;
+  text-align: center;
+  width: $hit-area;
+
+  &.facebook {
+    background-image: url("../img/social-icons/png/facebook25.png");
+    background-image: url("../img/social-icons/svg/facebook25.svg");
+    background-size: auto 3rem;
```

Comment:
I'm not sure I'm following. Would this establish Facebook as being a "default" social link - then the others as add-ons?

I believe this is moot if we decide to go with the suggestion of using @extend from the comment in footers.md, because it would separate the rules entirely.